android/lib/src/main/java/com/freshplanet/ane/AirInAppPurchase/billingManager/BillingManagerV5.java

```diff
@@ -125,17 +127,22 @@ public void onGetProductInfoFinishedListener(List<ProductDetails> productDetails
 
                             List<QueryProductDetailsParams.Product> subList = new ArrayList<>();
 
-                            for (String productId : skuSubsList) {
-                                subList.add(QueryProductDetailsParams.Product.newBuilder()
-                                        .setProductId(productId)
-                                        .setProductType(BillingClient.ProductType.SUBS)
-                                        .build());
-
+                            if(skuSubsList != null && skuSubsList.size() > 0) {
+                                for (String productId : skuSubsList) {
+                                    subList.add(QueryProductDetailsParams.Product.newBuilder()
+                                            .setProductId(productId)
+                                            .setProductType(BillingClient.ProductType.SUBS)
+                                            .build());
+                                            
+                                }
                             }
 
-                            QueryProductDetailsParams subParams = QueryProductDetailsParams.newBuilder()
-                                    .setProductList(subList)
-                                    .build();
+                            QueryProductDetailsParams subParams = null;
+                            if(subList.size()>0) {
```

Comment:
Please change to `if(subList.size() > 0)`

Reply:
I see only space difference in the code here and I fixed it